src/vivarium_public_health/risks/effect.py

```diff
@@ -112,8 +112,13 @@ def __init__(self, risk: str, target: str):
     def setup(self, builder: Builder) -> None:
         self.exposure = self.get_risk_exposure(builder)
 
-        self.target_modifier = self.get_target_modifier(builder)
-
+        self.relative_risk = self.get_relative_risk(builder)
```

Comment:
`self.relative_risk` should remain the Pipeline, since that's the real public entity. This method should be called `get_relative_risk_source()`, and you should assign it to an attribute called `self._relative_risk_source`.